third_party/README

```diff
@@ -138,3 +138,10 @@ Patches
 
  * Suppress CMake warning about policy CMP0048
  * Suppress tr1 namespace deprecation warnings
+
+ RapidJSON
+ ---------
+
+ MIT License
+
+ https://github.com/Tencent/rapidjson/tree/version1.1.0
```

Comment:
I would prefer the same style as above: Pointing to the release page and state the version number in a separate line below.

Reply:
Updated to match existing style

Reply:
@saschazelzer Could you please take a look at the latest changes and provide your feedback. I have other changes I want to publish, which depend on this PR.